tpkg/nodb_write.tdir/nodb_write.test

```diff
@@ -17,8 +17,9 @@ echo "export TPKG_PORT=$TPKG_PORT" >> .tpkg.var.test
 echo "export TPKG_NSD_PID=$TPKG_NSD_PID" >> .tpkg.var.test
 echo port: $TPKG_PORT
 
-$TPKG_NSD -c nsd_1.conf -u $LOGNAME -p $TPKG_PORT -P $TPKG_NSD_PID -V 5
+$TPKG_NSD -c nsd_1.conf -u $LOGNAME -p $TPKG_PORT -P $TPKG_NSD_PID -L 5 -F 0xFFFF
 wait_nsd_up nsd.log
+wait_logfile nsd.log "zonefiles write timer (nothing)" 5
```

Comment:
This test now needs the debug output to be available, like ixfr_retry.tdir test for it?